rtf.js

```diff
@@ -31,12 +31,18 @@ if (typeof RTFJS === "undefined") {
 			this.message = message;
 			this.stack = (new Error()).stack;
 		},
-		loggingEnabled: true,
```

Comment:
Please leave the default as is, if someone doesn't want logging they can set it to false after creating the RTFJS object.